sdk/storage/azure-storage-blobs/test/ut/recordings/BlobServiceClientTest.SetProperties.json

```diff
@@ -2,87 +2,86 @@
   "networkCallRecords": [
     {
       "Headers": {
-        "user-agent": "azsdk-cpp-storage-blobs/12.5.0-beta.2 (Windows 10 Pro 6.3 19044 19041.1.amd64fre.vb_release.191206-1406)",
-        "x-ms-client-request-id": "2a77ceea-064a-429a-6021-4ca52875ace4",
-        "x-ms-version": "2021-04-10"
+        "user-agent": "azsdk-cpp-storage-blobs/12.7.0-beta.1 (Windows 10 Enterprise 6.3 22621 22621.1.amd64fre.ni_release.220506-1250)",
```

Comment:
Because this test validate ApiVersion and using following code:
```
    properties.DefaultServiceVersion = Blobs::_detail::ApiVersion;
```
So if don't update, we can't pass this test like following error:

>  error: Expected equality of these values:
  downloadedProperties.DefaultServiceVersion.Value()
    Which is: "2021-04-10"
  properties.DefaultServiceVersion.Value()
    Which is: "2021-12-02